.spec/9378-full-sync-utf16-session/context.md

```
@@ -0,0 +1,103 @@
+- `capabilities.rs` unit tests: `initialize_prefers_first_supported_position_encoding`
+  (selects UTF-8), `initialize_accepts_utf16_when_it_is_first_supported_position_encoding`,
+  `initialize_falls_back_to_utf16_when_position_encodings_have_no_supported_values`;
+- `crates/perl-lsp-rs/tests/lsp_3_17_lifecycle_tests.rs`
+  `test_position_encoding_advertised_is_clamped_to_utf16_pending_phase_2`;
+- `effective_surface_parity_tests.rs`
+  `pull_diagnostic_client_with_refresh_supports_and_utf8_preference_matches`
+  (subject offers `["utf-32", "utf-8"]`).
+
+## Branch-selection gate
+
+#8129 selected `full_document_utf16` for the v0.18 release candidate and
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)